common/test/acceptance/tests/lms/test_certificate_web_view.py

```diff
@@ -225,7 +225,7 @@ def complete_course_problems(self):
         self.course_nav.q(css='fieldset label:nth-child(3) input').nth(1).click()
 
         # Submit the answer
-        self.course_nav.q(css='button.check.Check').click()
+        self.course_nav.q(css='button.check.Submit').click()
```

Comment:
Ah yes-- I didn't think that the CSS classes had changed. But I should have had you run the test locally, which might have helped to isolate the issue.

But @alisan617, why is there a css class of "Submit" at all? If the "Submit" button is the only one with the class "check", I recommend dropping the "Submit" class entirely, as it is not our style to have CSS classes that match (English) UI text. I would think that "button.check" is sufficient.